docs/source/reference/index.rst

```diff
@@ -19,3 +19,69 @@ All pods can run as non-root, with two exceptions:
+| backend-database         | backend-server, backend-worker, backend-api-events, backend-worker-events, backend-scheduler, backend-scheduler-worker, job-migrations  | NONE                                                                                                                                |
++--------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| backend-redis            | backend-server, backend-worker, backend-builder, backend-api-events, backend-worker-events, backend-scheduler, backend-scheduler-worker | NONE                                                                                                                                |
++--------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| backend-object-storage   | backend-server, backend-builder, backend-worker                                                                                         | NONE                                                                                                                                |
++--------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| backend-docker-registry  | backend-task-build, backend-builder, backend-worker, backend-scheduler, backend-scheduler-worker                                        | NONE                                                                                                                                |
++--------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| backend-server           | backend-worker, backend-builder, backend-server (other orgs)                                                                            | orchestrator-server, backend-server (other orgs), backend-database, backend-redis, backend-object-storage, backend-docker-registry, |
```

Comment:
I don't know if it works saying internet as egress

Reply:
how would you phrase it?